core/chaincode/shim/chaincode.go

```diff
@@ -252,39 +264,39 @@ func (stub *ChaincodeStub) init(uuid string, secContext *pb.ChaincodeSecurityCon
 
 // ------------- Call Chaincode functions ---------------
 
-// InvokeChaincode function can be invoked by a chaincode to execute another chaincode.
+// InvokeChaincode locally calls the specified chaincode `Invoke`.
 func (stub *ChaincodeStub) InvokeChaincode(chaincodeName string, function string, args []string) ([]byte, error) {
 	return handler.handleInvokeChaincode(chaincodeName, function, args, stub.UUID)
 }
 
-// QueryChaincode function can be invoked by a chaincode to query another chaincode.
+// QueryChaincode locally calls the specified chaincode `Query`.
```

Comment:
People always ask me if this creates a second transaction. Should we state that it does not in the comment?